staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -173,6 +173,7 @@ func decorateResponseWriter(ctx context.Context, responseWriter http.ResponseWri
 
 	// check if the ResponseWriter we're wrapping is the fancy one we need
 	// or if the basic is sufficient
+	//lint:ignore SA1019 backwards compatibility
```

Comment:
The change looks right in the sense that we cannot drop CloseNotifier while we still have vendored dependencies that use it https://github.com/kubernetes/kubernetes/issues/76469#issuecomment-483676874. But I'm not sure we want to ignore this issue in the static check. Shouldn't we keep the warning as a reminder to eventually fix it, or even have a plan to fix the upstream vendors to unblock ourselves?

Reply:
Thanks for code review! Yeah you are right, I see most of ths `CloseNotifier` problems exist in `apiserver` package and there is another closed issue [#85408](https://github.com/kubernetes/kubernetes/issues/85408) fixed some of them. I will follow up on these problems if the community has a plan to fix them. 

I will remove `lint:ignore` comment in another commit.

Reply:
Great. Thanks!